models/Item.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require("mongoose");
+
+const itemSchema = new mongoose.Schema({
+    // var name below TBC
+    postID: {
+        type: Number,
+        required: true,
+    },
+    type: {
+        type: ["Request", "Offer"],
+        required: true,
+    },
+    title: {
+        type: String,
+        required: true,
+        max: 200,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    category: {
+        type: String,
+        required: true,
+    },
+    images: String,
+    delivery: {
+        type: ["Yes", "No"],
```

Comment:
Yes/No is a little ambiguous as it is interpreted differently depending on whether the post is a request or offer. Suggest to just leave this as an open-ended String for now? or enum: ["required", "not required", "included", "not included"]